frontend/app/(footer)/fulfilment/[fulfilmentSessionId]/[fulfilmentEntityId]/page.tsx

```
@@ -0,0 +1,135 @@
+      if (getFulfilmentEntityInfoResponse.url === null) {
+        fulfilmentSessionEntityPageLogger.error(
+          `Stripe Fulfilment Entity URL is null when it should not be, fulfilmentSessionId: ${
+            params.fulfilmentSessionId
+          }, fulfilmentEntityId: ${params.fulfilmentEntityId}, getFulfilmentEntityInfoResponse: ${JSON.stringify(
+            getFulfilmentEntityInfoResponse
+          )}`
+        );
+        router.push("/error");
+        return;
+      }
+      router.push(getFulfilmentEntityInfoResponse.url);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)